src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,26 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard/MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div key={Number(movie.imdbId.replace('tt', ''))} className="card">
+        <MovieCard {...movie} />
+      </div>
```

Comment:
add key directly to the MovieCard
also add just movie.imdbId, tt at the beginning is perfectly fine